wp-content/plugins/hurumap/package.json

```diff
@@ -30,11 +30,13 @@
   },
   "scripts": {
     "lint": "yarn eslint --fix  './src'",
-    "start": "concurrently \"yarn start:data-page\"  \"yarn start:blocks\"",
+    "start": "concurrently \"yarn start:definitions\"  \"yarn start:blocks\"",
     "start:blocks": "wp-scripts start ./src/FlourishBlock/index.js ./src/HURUmapBlock/index.js ./src/FeaturedData/index.js ./src/CardBlock/index.js --output-path=build/blocks",
-    "start:data-page": "wp-scripts start ./src/index.js --output-path=build/data-page",
-    "build": "yarn build:data-page && yarn build:blocks && yarn build:micro-frontend",
-    "build:data-page": "wp-scripts build ./src/index.js --output-path=build/data-page",
+    "start:definitions": "wp-scripts start ./src/ChartDefinition/index.js --output-path=build/definitions",
+    "start:sections": "wp-scripts start ./src/ChartsSection/index.js --output-path=build/sections",
+    "build": "yarn build:definitions && yarn build:blocks && yarn build:micro-frontend",
+    "build:definitions": "wp-scripts build ./src/ChartDefinition/index.js --output-path=build/definition",
+    "build:sections": "wp-scripts build ./src/ChartsSection/index.js --output-path=build/section",
```

Comment:
Why are `start:sections` & `build:sections` excluded from `start` and `build` commands respectively?